From: Modestas Vainius <modax@debian.org>
Subject: Disable tests which require active DBus
Origin: vendor
Forwarded: not-needed
Last-Update: 2012-05-12

The following tests need active DBus daemon at the moment. Disable them.

- akonadi-imapstreamparsertest
- akonadi-akdbustest
- akonadi-akstandarddirstest
- akonadi-handlertest
- akonadi-dbconfigtest

--- a/server/tests/unittest/CMakeLists.txt
+++ b/server/tests/unittest/CMakeLists.txt
@@ -18,13 +18,13 @@ macro(add_server_test _source _libs)
   endif (AKONADI_STATIC_SQLITE)
 endmacro(add_server_test)
 
-add_server_test(imapstreamparsertest.cpp akonadiprivate)
+#add_server_test(imapstreamparsertest.cpp akonadiprivate)
 add_server_test(scopetest.cpp akonadiprivate)
 add_server_test(handlerhelpertest.cpp akonadiprivate)
 add_server_test(querybuildertest.cpp akonadiprivate)
 add_server_test(dbinitializertest.cpp akonadiprivate)
 add_server_test(dbupdatertest.cpp akonadiprivate)
-add_server_test(akdbustest.cpp akonadiprivate)
-add_server_test(akstandarddirstest.cpp akonadiprivate)
-add_server_test(handlertest.cpp akonadiprivate)
-add_server_test(dbconfigtest.cpp akonadiprivate)
+#add_server_test(akdbustest.cpp akonadiprivate)
+#add_server_test(akstandarddirstest.cpp akonadiprivate)
+#add_server_test(handlertest.cpp akonadiprivate)
+#add_server_test(dbconfigtest.cpp akonadiprivate)
